angular-app/src/app/discovery/discovery.component.html

```diff
@@ -2,38 +2,58 @@
   <span>Discover</span>
 </mat-toolbar>
 
-<mat-selection-list #tags>
-  <mat-list-option *ngFor="let tag of tagOptions">
-    {{tag}}
-  </mat-list-option>
-</mat-selection-list>
+<div class="filter-chip">
+        <mat-chip class="pi-chip-filter"
+                  [matMenuTriggerFor]="longListFilter"
+                  [selected]="selectedLongListTags.length > 0">
+            {{getMultiSelectChipName(selectedLongListTags) || "Search filter"}}
+            <mat-icon matChipRemove
+                      (click)="selectedLongListTags = clearSelectedTags()"
```

Comment:
This seems like it will only show one chip and not the list of chips. I would expect to see a ngFor creating each chip and to use `mat-chip-list`  to hold each chip. https://material.angular.io/components/chips/overview

Am I reading this code wrong?

Reply:
So the chip is like a toggle. When clicked, the selection list of tags is supposed to be displayed. Right now, all it is displaying is the input where you can type tags you want to search. If you click on the example I linked, you can get a better idea of it.

Reply:
It looks like the actual list is down at line 44

Reply:
I see thanks.